apps/web/public/static/locales/en/common.json

```diff
@@ -121,6 +121,9 @@
   "team_upgrade_banner_description": "Thank you for trialing our new team plan. We noticed your team \"{{teamName}}\" needs to be upgraded.",
   "team_upgrade_banner_action": "Upgrade here",
   "team_upgraded_successfully": "Your team was upgraded successfully!",
+  "org_upgrade_banner_description": "Thank you for trialing our Organization plan. We noticed your Organization \"{{teamName}}\" needs to be upgraded.",
+  "org_upgrade_banner_action": "Upgrade here",
```

Comment:
we can just reuse `team_upgrade_banner_action` instead of adding a new translation